pkg/serverless/trace/inferredspan/constants.go

```diff
@@ -11,9 +11,11 @@ const (
 	apiName          = "apiname"
 	bucketARN        = "bucket_arn"
 	bucketName       = "bucketname"
+	busName          = "x-datadog-bus-name"
```

Comment:
I'm not sure this tags should be here as this are like headers? 🤔  also, what about `x-datadog-resource-name`? i wonder if `bus-name` is going to be quite niche

Reply:
I noticed `sentTimestamp` was used by the SQS implementation, and it's located in this same file. I'm happy to move it if it doesn't make sense, but it is indeed a constant used for inferred spans, so I think it makes sense here. What do you think?

I agree that `x-datadog-resource-name` would be a better variable name! Gonna change that